src/mappings/control/events/memberRemoved.ts

```diff
@@ -0,0 +1,25 @@
+import { Context, Block } from '../../../processor'
+import { Event } from '../../../types/generated/support'
+
+import { getMemberRemovedData } from './getters'
+import { getOrgMember } from '../../util/db/getters'
+
+import { arrayToHexString, addressCodec } from '../../util/helpers'
+import { ObjectNotExistsWarn } from '../../../common/errors'
+
+
+async function handleMemberRemovedEvent(ctx: Context, block: Block, event: Event, name: string) {
+	const eventData = getMemberRemovedData(ctx, event)
+	let address = addressCodec.encode(eventData.who)
+	let orgId = arrayToHexString(eventData.orgId)
+
+	let member = await getOrgMember(ctx.store, orgId, address)
+	if (!member) {
+		ctx.log.warn(ObjectNotExistsWarn(name, 'Member', `${orgId}-${address}`.toLowerCase()))
+		return
+	}
+
+	await ctx.store.remove(member)
+}
+
+export { handleMemberRemovedEvent }
```

Comment:
-> we could also need a handleMemberStateChanged Event in general, so user can receive notificcation for being accepted.